eth2.0-implementers-calls/call_041.md

```diff
@@ -0,0 +1,448 @@
+# Ethereum 2.0 Implementers Call 41 Notes
+
+### Meeting Date/Time: Thursday June 11, 2020 at 14:00 UTC
+### Meeting Duration:  1 hr
+### [GitHub Agenda](https://github.com/ethereum/eth2.0-pm/issues/158)
+### [Audio/Video of the meeting](https://youtu.be/WmU3k2v4UA8)
+### Moderator: Danny Ryan
+### Notes: Pooja Ranjan
+
+-----------------------------
+
+# Contents
+
+- [1. Testing and Release Updates](#1-testing-and-release-updates)   
+- [2. Testnets](#2-testnets)   
+    - [2.1 Validator key hygiene](#21-validator-key-hygiene)
+    - [2.2 Wallet vs. keystore discussion](#22-wallet-vs-keystore-discussion)
+- [3. Client Updates](#3-client-updates)   
+   - [3.1 Teku](#31-teku)  
+   - [3.2 Nimbus](#32-nimbus)   
+   - [3.3 Lodestar](#33-lodestar)   
+   - [3.4 Lighthouse](#34-lighthouse)   
+   - [3.5 Prysm](#35-prysm)   
+   - [3.6 Nethermind](#36-nethermind)   
+   - [3.7 Trinity](#37-trinity)  
+- [4. API WG update](#4-api-wg-update)
+- [5. Research Updates](#5-research-updates)   
+   - [5.1 Verkle trees](#51-verkle-trees)   
+- [6. Networking](#6-networking)   
+- [7. Spec Discussion](#7-spec-discussion)   
+- [8. Open Discussion/Closing Remarks](#8-open-discussionclosing-remarks)   
+   
+
+-----------------------------
+
+# 1. Testing and Release Updates
+
+Video | [01:40](https://youtu.be/WmU3k2v4UA8?t=100)
+-|-
+
+**Danny**: There are a couple of non-substantive but clarifying and potentially optimizing PR in the Fork choice, that are up right now. I expect them to go to development soon, and will make a note to the team. Probably release will be in next week or so. 
+
+Lighthouse has integrated Alex's fork choice tests. We are still looking at and figuring how to officially release them.
+
+Other testing updates?
+
+**Proto**: 
+
+### Testnet
+
+For testnet, I've worked on [validator tooling](https://github.com/protolambda/eth2-val-tools) for  wallets for stress testing. Testnet tooling involved, so be warned for bugs. 
+
+Lighthouse assumes strong point to use key store specifically. It will bring some change to Serenity. Worked on [E2DB](https://github.com/protolambda/e2db/). Lighthouse has API exposed to fork choice information and blocks. This tool can be helpful in analytics. 
+
+**Danny**: Other testing update?
+
+**Mehdi**: 
```

Comment:
What did he say?

Reply:
Same with Mehdi's testnet update. It is added under the subhead "Beacon Fuzz Update" (58-69).